.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -24,7 +24,6 @@ t0:
   - container_hardening/test_container_hardening.py
   - database/test_db_config.py
   - database/test_db_scripts.py
-  - dhcp_relay/test_dhcp_pkt_recv.py
```

Comment:
Why remove this script from the t0 scenario?
DHCP relay is a critical feature of T0.

Reply:
Because in this script, there is a mark pytest.mark.topology('mx'), so I suppose it should only be run on mx

Reply:
I will use another PR to fix this